sdks/python/apache_beam/runners/dataflow/dataflow_exercise_streaming_metrics_pipeline_test.py

```diff
@@ -113,7 +113,7 @@ def run_pipeline(self):
     return dataflow_exercise_streaming_metrics_pipeline.run(argv)
 
   # Need not run streaming test in batch mode.
-  @attr('IT', 'ValidatesRunner', 'sickbay-batch')
+  @attr('IT')
```

Comment:
Please change this to:
```
@pytest.mark.it_validatesrunner
@pytest.mark.no_sickbay_batch
```

I want to remove the `@attr('IT')` so it only runs in VR test suites.

Reply:
Done!